bokeh/models/dom.py

```diff
@@ -142,6 +147,28 @@ class Placeholder(DOMNode):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
+class ValueOf(Placeholder):
+    """ """
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+    obj: HasProps = Required(Instance(HasProps), help="""
+    The object whose property will be observed.
```

Comment:
```suggestion
    The object to observe a property on.
```
A nit, surely, but the antecedent of "whose" should be a person.

Reply:
Not according to https://www.merriam-webster.com/words-at-play/whose-used-for-inanimate-objects.